cmd/lib_linux.go

```diff
@@ -24,21 +24,21 @@ func GetHost(ctx context.Context) (host.Host, error) {
 		if err != nil {
 			return nil, err
 		}
-	} else if dockerContainer != "" {
-		hst, err = ihost.NewDocker(ctx, dockerContainer, dockerUser)
+	} else if docker != "" {
+		hst, err = ihost.NewDocker(ctx, docker)
 		if err != nil {
 			return nil, err
 		}
 	} else {
-		return nil, errors.New("no target host specified: must pass either --localhost, --ssh or --docker-container")
+		return nil, errors.New("no target host specified: must pass either --target-localhost, --target-ssh or --target-docker")
 	}
 
 	return wrapHost(ctx, hst)
 }
 
 func AddHostFlags(cmd *cobra.Command) {
 	cmd.Flags().BoolVarP(
-		&localhost, "localhost", "", defaultLocalhost,
+		&localhost, "target-localhost", "1", defaultLocalhost,
```

Comment:
This short-hand seems quite odd IMO, but I am not sure what would be the best approach here, maybe leave it without a short-hand?

Reply:
TBH, IDK. These can me

Reply:
I had to unresolve, sorry, I'm sure you've seen this, lol: https://github.com/ltworf/funny-manpages/blob/master/man/sex.6fun#L16.

Reply:
It is indeed a funny page, but in a serious note you can find similar approaches in linux basic commands. 
https://man7.org/linux/man-pages/man1/ls.1.html

Reply:
Yes I know, I liked your solution. But `--target-localhost` sort feels like `sex -1`, lol.